components/schema-migrator/migrations/director/202012081537_ord_service_initial.up.sql

```diff
@@ -22,8 +22,8 @@ $$
 BEGIN
     NEW.release_status := CASE
                               WHEN NEW.release_status IS NOT NULL THEN NEW.release_status
-                              WHEN NEW.version_deprecated IS NULL AND NEW.version_for_removal THEN 'decommissioned'
                               WHEN NEW.version_deprecated THEN 'deprecated'
+                              WHEN NEW.version_for_removal THEN 'decommissioned'
```

Comment:
decommissioned should be with higher priority here because if an API is removed it is always deprecated before that and in that case, we will have both true and the realease_status will be deprecated, but the API is actually decommissioned.